starknet_py/net/account/account_client.py

```diff
@@ -593,7 +587,9 @@ async def _verify_message_hash(self, msg_hash: int, signature: List[int]) -> boo
             raise ex
 
 
-def add_signature_to_transaction(tx: Invoke, signature: List[int]) -> Invoke:
+def add_signature_to_transaction(
+    tx: Union[Invoke, Declare, DeployAccount], signature: List[int]
```

Comment:
So `tx` is just AccountTransaction or?

Reply:
There also is a new Type `SignableTransaction` in `account.py`. Is not every account transaction signable?